boards/common/nrf52xxxdk/Makefile.include

```diff
@@ -24,9 +24,9 @@ export JLINK_DEVICE := nrf52
 include $(RIOTBOARD)/$(BOARD)/Makefile.dep
 
 # special options when using SoftDevice
-ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+ifneq (,$(filter nordic_softdevice_ble nordic_softdevice_sixlowpan,$(USEPKG) $(USEMODULE)))
```

Comment:
If `nordic_softdevice_ble` is deprecated, please remove.

Reply:
Nope, it is not deprecated.  The package has now been split into to a package proper and a module inside the package.  The package itself brings in the softdevice and BLE, including the possibility of using the BLE GATT APIs from the SD.  The module adds 6lowpan glue on the top of that.

I needed to separate them so that I can compile binaries that want only GATT and not 6lowpan.  Without the split there was an implicit dependency that brought in the whole TCP/IP stack into the binary, even though it was not needed.